datasync/chaindatafetcher/kas/repository.go

```diff
@@ -88,3 +97,54 @@ func (r *repository) SetComponent(component interface{}) {
 		r.blockchainApi = c
 	}
 }
+
+func makeEOAListStr(eoaList map[common.Address]struct{}) string {
+	eoaStrList := ""
+	for key := range eoaList {
+		eoaStrList += "\""
+		eoaStrList += strings.ToLower(key.String())
+		eoaStrList += "\""
+		eoaStrList += ","
+	}
+	return eoaStrList[:len(eoaStrList)-1]
+}
+
+func (r *repository) InvalidateCacheEOAList(eoaList map[common.Address]struct{}) {
```

Comment:
How about leaving a log(Debug or Trace would be enough) or collecting a metric about the number of EOAs invalidated from the cache?

Reply:
I left a log message instead of collecting metrics. d56ee56156b905a2ebba88580cb85b3d44fe9571